Server/src/controllers/mentors.controller.js

```diff
@@ -15,7 +15,7 @@ class MentorController {
       const { isadmin, ...data } = findMentors[0];
       return res.status(200).json({ status: 200, data });
     } catch (err) {
-      return res.status(400).json({status: 400, error: err.message});
+      return res.status(400).json({status: 400, message: 'mentors retrieved succefully', error: err.message});
```

Comment:
You don't need to return a message on error. status and error are just enough

Reply:
Ohh yeah I going to correct it as fast as possible